spec/values/calculation/round.hrx

```diff
@@ -0,0 +1,371 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `round()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> error/syntax/no_arg/input.scss
+a {b: round()}
+
+<===> error/syntax/no_arg/error
+Error: Missing argument $number.
+  ,--> input.scss
+1 | a {b: round()}
+  |       ^^^^^^^ invocation
+  '
+  ,--> sass:math
+1 | @function round($number) {
+  |           ============== declaration
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/two_args/compatibility/input.scss
+a {b: round(3.4px + 1px, 1 + 4)}
+
+<===> error/syntax/two_args/compatibility/error
+Error: 4.4px and 5 are incompatible.
+  ,
+1 | a {b: round(3.4px + 1px, 1 + 4)}
+  |             ^^^^^^^^^^^ 4.4px
+  |                          ===== 5
+  '
+  input.scss 1:13  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/two_args/wrong_arg/input.scss
+a {b: round(nearest, 5)}
+
+<===> error/syntax/two_args/wrong_arg/error
+Error: If strategy is not null, step is required.
+  ,
+1 | a {b: round(nearest, 5)}
+  |       ^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/strategy/number/input.scss
+a {b: round(3px, 3px, 2px)}
+
+<===> error/syntax/strategy/number/error
+Error: 3px must be either nearest, up, down or to-zero.
+  ,
+1 | a {b: round(3px, 3px, 2px)}
+  |       ^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> perserved/unknown_variable/number/input.scss
```

Comment:
Typo: preserved, not perserved

Reply:
Thanks, found this mistake in the min.hrx tests too, fixed it. :)